tests/functional/aws-node-sdk/test/bucket/deleteBucketTagging.js

```diff
@@ -0,0 +1,101 @@
+const assert = require('assert');
+const { errors } = require('arsenal');
+const { S3 } = require('aws-sdk');
+const async = require('async');
+
+const getConfig = require('../support/config');
+
+const bucket = 'policyputtaggingtestbucket';
+
+const validTagging = {
+    TagSet: [
+        {
+            Key: 'key1',
+            Value: 'value1',
+        },
+        {
+            Key: 'key2',
+            Value: 'value2',
+        },
+    ],
+};
+
+// Check for the expected error response code and status code.
+function assertError(err, expectedErr, cb) {
+    if (expectedErr === null) {
+        assert.strictEqual(err, null, `expected no error but got '${err}'`);
+    } else {
+        assert.strictEqual(err.code, expectedErr, 'incorrect error response ' +
+            `code: should be '${expectedErr}' but got '${err.code}'`);
+        assert.strictEqual(err.statusCode, errors[expectedErr].code,
+            'incorrect error status code: should be  ' +
+            `${errors[expectedErr].code}, but got '${err.statusCode}'`);
+    }
+    cb();
```

Comment:
This callback seems completely separate from the rest of the function and as nothing is async doesn't seem required.

Reply:
done [tests/utilities/bucketTagging-util.js](https://github.com/scality/cloudserver/pull/4481/commits/96157ec37c9e4398618eb993ff7e5fb92c4e8141#diff-5f6289713c22c427fbe9c6a9db3190cf73c0637ebda0a9a92f8c660ce9ac6ac1R1)

Reply:
I think you linked the this in the wrong comment